packages/hash/api/src/model/knowledge/orgMembership.model.ts

```diff
@@ -84,7 +84,7 @@ export default class extends LinkEntityModel {
     };
 
     const entity = await user.createOutgoingLink(graphApi, {
-      ownedById: systemAccountId,
+      ownedById: systemOrgAccountId,
```

Comment:
This feels like a bug in the previous code. Fairly certain we want this to be owned by the org's accountId, not the _system_ org. Might not be the same here

Reply:
Agreed, have changed it in https://github.com/hashintel/hash/pull/1596/commits/ba5b6ccf460bee98b8b821be7244b007726609bf